crates/turborepo-ui/src/tui/input.rs

```diff
@@ -2,19 +2,15 @@ use std::time::Duration;
 
 use crossterm::event::{KeyCode, KeyEvent, KeyEventKind, KeyModifiers};
 
-use super::{event::Event, Error};
+use super::{app::LayoutSections, event::Event, Error};
 
-#[derive(Debug, Clone, Copy)]
```

Comment:
We should add these back and change the functions in the file to take ownership instead of references. It will require adding `#[derive(Debug, Clone, Copy)]` to the definition of `LayoutSections`